### DISCUSSION TITLE
can i put this on scratchsuite upptime?

### DISCUSSION THREAD

**Discussion by FunctionalMetatable** [FlagClicked/FlagClicked#40] [Help] [answered]
**.**

**Comment by danielgallab** [comment, marked as answer]
Please explain what you would like to do... you would like to take the source code? the website? please explain.

**Reply by FunctionalMetatable** [reply]
[ScratchSuite](//scratchsuite.github.io/scratchrs)  has a uptime checker which pings websites every five minutes to see if they are up or not. I'd like to add   whenflagclicked.org to it. Is that ok?

**Reply by danielgallab** [reply]
We already have one... https://status.whenflagclicked.org

**Reply by FunctionalMetatable** [reply]
ok
